### PR TITLE
Add docs for CDN

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: docs/cdn.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+* Service: `theforeman.org`
+* Domains: `theforeman.org` and `www.theforeman.org`
+* Backend: `web02.theforeman.org`, with TLS enabled and a health check for `HEAD /introduction.html`
+* this service currently gets no traffic as it is not configured in DNS
+
+#### downloads.theforeman.org
+
+* Service: `downloads.theforeman.org`
+* Domains: `downloads.theforeman.org`
+* Backend: `web02.theforeman.org`, with TLS enabled and a health check for `HEAD /HEADER.html`
```
Comment (ekohl) [inline docs/cdn.md:33]:
```suggestion
* Backend: `web01.osuosl.theforeman.org`, with TLS enabled and a health check for `HEAD /HEADER.html`
```

(repeated for all below)

File: docs/cdn.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+## Which vhosts are served via CDN?
+
+* downloads.theforeman.org
+* stagingdeb.theforeman.org
+
+## Setup
+
+### Varnish
+
+#### theforeman.org
```
Comment (ekohl) [inline docs/cdn.md:22]: This isn't actually served by the CDN

File: docs/cdn.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+We use [Fastly](https://www.fastly.com) as a CDN provider for our web content.
+
+## Who has access?
+
+* Greg
+* Ewoud
+* Evgeni
+
+## Which vhosts are served via CDN?
```
Comment (ekohl) [inline docs/cdn.md:13]: Missing yum.t.o and deb.t.o

File: docs/cdn.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+Fastly provides a shared certificate which has `theforeman.org` and `*.theforeman.org` added. (There is a `_globalsign-domain-verification` `TXT` record in the theforeman.org DNS zone for that.)
+
+### DNS
+
+Each vhost needs a CNAME pointing at `p2.shared.global.fastly.net` or `dualstack.p2.shared.global.fastly.net` for a DualStack setup.
+
+## TODO
+
+* Investigate logging posibilities
+* Move more vhosts as soon as the current ones are deemed stable
```
Comment (ekohl) [inline docs/cdn.md:52]: I think the only theforeman.org itself is not fronted by the CDN now (since 060215525531dded966f79846c369260333fe6b8 added deb).

**Review by ekohl — CHANGES_REQUESTED** [review]

**Comment by ekohl** [comment]
Oh, and it should point to `ansible/fastly.yml` for the actual config. Also wondering if https://github.com/theforeman/foreman-infra/blob/master/docs/webserver.md doesn't document the CDN already.

**Comment by ehelms** [comment]
I took this from Redmine wiki, looks to @evgeni on he wants me to treat this

**Comment by evgeni** [comment]
Nuke it, webserver.md should have most details

**Pull request closed without merging**